### PR TITLE
{tools}[system/system] rpmrebuild v2.18

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/925db4e4e068820af9b0ce518e169a40 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/r/rpmrebuild/rpmrebuild-2.18.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+homepage = 'http://rpmrebuild.sourceforge.net/'
+description = """rpmrebuild is a tool to build an RPM file from a package that has already been
+installed in a basic use"""
+
+toolchain = SYSTEM
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
```
Comment (branfosj) [inline easybuild/easyconfigs/r/rpmrebuild/rpmrebuild-2.18.eb:14]:
```suggestion
checksums = ['919ca6252eafbfb58dd54b0efa3ca4e6da293364937dcfe32c32a35b238a5837']

```

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0105u03b - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/1d1379b716014f6916b960f902cf7975 for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3107.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/8445b150e48598e0102a5165c400f2c8 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @Micket!

**Pull request merged**